cmd/adhoc/cmd/download.go

```diff
@@ -31,16 +50,40 @@ func init() {
 		0,
 		"Download data from the last X years",
 	)
-
 	root.AddCommand(download)
 }
 
 func onRun(cmd *cobra.Command, args []string) error {
+	Month, _ := cmd.Flags().GetInt(cli.Months)
+	Year, _ := cmd.Flags().GetInt(cli.Years)
+	cliDay, err := cmd.Flags().GetInt(cli.Days)
+
+	if cliDay < 2 {
+		logrus.Error("No data available.")
+		logrus.Error("Try 3 days or greater.")
```

Comment:
Minor thing, but shouldn't this 3 be 2? Since the minimum (and default) amount of days is 2

Reply:
No, because Syncron actually counts today too, so it's for today, yersterday and day before yesterday.

Reply:
This is confusing. Does setting '--days' to 2 mean today, yesterday and the day before?

Reply:
Yes, as the tool suggests, you are asking syncron to download from the past "x" days, so you are asking for the past 2 days (yesterday and day before yesterday) as well as today.

Reply:
But it says "from the last X days", which for 2 is today and yesterday. I don't know, I saw it easier to understand before.

Reply:
My logic is telling me last 2 days means those days are in the past, and then today, the present.
I have no strong opinion on this, I can make the change and move on.